### ISSUE TITLE
Add avro 1.9 support

### ISSUE THREAD

**Issue by regadas** [spotify/scio#1949]
Initial work has been done here #1946.

Blocked by `Apache Beam` https://issues.apache.org/jira/browse/BEAM-8388.

**Comment by jto** [comment]
I think we need to use the same version of Avro as Beam since `1.8.x` and `1.9.x` are not compatible. See https://issues.apache.org/jira/browse/BEAM-7328

**Comment by jto** [comment]
@regadas seems like this will never happen. Close the issue ?

**Comment by regadas** [comment]
The above jira ticket was superseded by https://issues.apache.org/jira/browse/BEAM-8388.

i can close or we can just add a blocked label here to track things.
